Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 gemspec
 gem 'codeclimate-test-reporter', group: :test, require: nil
+gem 'ruby_event_store', github: 'arkency/ruby_event_store'
```

Comment:
of course I will move it back to the gemspec once the newest Ruby Event Store has been released to rubygems